src/compiler/scala/tools/nsc/backend/jvm/BTypes.scala

```diff
@@ -938,13 +912,14 @@ abstract class BTypes {
     })
 
     /**
-     * The prefix of the internal name until the last '/', or the empty string.
+     * packageInternalName - The prefix of the internal name until the last '/', or the empty string.
+     * simpleName - The class name without the package prefix
      */
-    def packageInternalName: String = {
+    lazy val (packageInternalName:String, simpleName: String) = {
```

Comment:
now there is a feature that I was not aware of. reasonable easy to work around it though